I'm sorry, but an uncaught exception occurred.

While running game code:
  File "characters.rpy", line 399, in script
    image msG curious explaining = LiveComposite(
  File "characters.rpy", line 405, in <module>
    (0, 0), "msG eyes normal",
TypeError: 'unicode' object is not callable

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "characters.rpy", line 399, in script
    image msG curious explaining = LiveComposite(
  File "/home/guichristmann/Programming/Ren'Py/renpy-7.1.3-sdk/renpy/ast.py", line 980, in execute
    img = renpy.python.py_eval_bytecode(self.code.bytecode)
  File "/home/guichristmann/Programming/Ren'Py/renpy-7.1.3-sdk/renpy/python.py", line 1937, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "characters.rpy", line 405, in <module>
    (0, 0), "msG eyes normal",
TypeError: 'unicode' object is not callable

Linux-4.15.0-47-generic-x86_64-with-debian-buster-sid
Ren'Py 7.1.3.1092
Lectures with Prof. JB 1.0
Wed Apr 10 10:13:41 2019
